mungegithub/mungers/release-note-label.go

```diff
@@ -125,8 +130,21 @@ func (r *ReleaseNoteLabel) Munge(obj *github.MungeObject) {
 		return
 	}
 
-	if !obj.HasLabel(releaseNoteLabelNeeded) {
-		obj.AddLabel(releaseNoteLabelNeeded)
+	labelToAdd := determineReleaseNoteLabel(obj)
+	if labelToAdd == releaseNoteLabelNeeded {
+		if !obj.HasLabel(releaseNoteLabelNeeded) {
+			obj.AddLabel(releaseNoteLabelNeeded)
+		}
+		if !obj.HasLabel(doNotMergeLabel) {
```

Comment:
@grodrigues3 Can we please make sure we don't add doNotMergelabel to all PRs that don't have the label? and rather gate on the label being ok?